src/frontend/csrs-portal/src/app/components/landing/landing.component.scss

```diff
@@ -0,0 +1,589 @@
+@import "palette";
+@import "bootstrap/bootstrap-imports";
+
+h1 {
+  font-size: 3em;
+  line-height: 1.1em;
+  margin-bottom: 50px;
+  font-weight: bold;
+  text-shadow: 2px 2px 4px #000;
+}
+
+h3 {
+  font-size: 1.875em;
+  line-height: 1.4em;
+  font-weight: bold;
+  margin-bottom: 10px;
+  color: theme-palette(blue);
+}
+
+h4 {
+  font-size: 1.5em;
+  font-weight: bold;
+  margin-bottom: 20px;
+  color: theme-palette(blue);
+}
+
+.lead {
+  font-size: 1.5em;
+  margin-bottom: 10px;
+  line-height: 1.5em;
+  text-shadow: 2px 2px 4px #000;
+}
+
+@media (max-width: 320px) {
```

Comment:
are you using these media queries in the project

Reply:
It was removed.

Reply:
Going forward I would appreciate if you specify the commit number here as well. We can find that out for sure but adding the commit # here could make the reviewers' lives easier. Thanks!

Reply:
It is partially removed not fully.